### PR TITLE
added ts-ignore

### PR DESCRIPTION
**Pull request by yourgirlcodes**

### CODE REVIEW

File: index.ts [hunk 1]
```diff
@@ -367,6 +367,7 @@ const sdkOptions: Partial<typeof CDP.DefaultOptions> = {
             let ingestResponses: Array<Partial<CDPErrorResponse>>;
             if (!context.delay) {
                 terminal.cyan(`Ingesting ${context.eventsNum * (Math.max(1, context.customersNum))} fake events\n`);
+                // @ts-ignore
```
Comment (iBaryo) [inline index.ts:370]: booo

Comment (yourgirlcodes) [reply index.ts]: killed

File: index.ts [hunk 2]
```diff
@@ -378,6 +379,7 @@ const sdkOptions: Partial<typeof CDP.DefaultOptions> = {
                 });
 
                 let i = 1;
+                // @ts-ignore
```
[the remark this answers is not in the document]
Comment (yourgirlcodes) [reply index.ts]: killed

File: boilerplate/Events/CloudStorage.ts [hunk 3]
```diff
@@ -12,12 +12,12 @@ interface EventConfig {
     }>>;
 }
 
-//TODO: fix CloudStorage Type
+//TODO: fix CloudStorage Type (schemaId---> make a 'WithSchemaId' in entities so that we can omit it here from Event)
 
 export const boilerplateCloudStorageEvent: EventConfig = {
+    //@ts-ignore
```
[the remark this answers is not in the document]
Comment (yourgirlcodes) [reply boilerplate/Events/CloudStorage.ts]: killed

**Pull request merged**